topic-operator/src/main/java/io/strimzi/operator/topic/v2/TopicOperatorConfig.java

```diff
@@ -239,6 +298,7 @@ public String toString() {
                 "\n\tsslEndpointIdentificationAlgorithm='" + sslEndpointIdentificationAlgorithm + '\'' +
                 "\n\tsaslEnabled=" + saslEnabled +
                 "\n\tsaslMechanism='" + saslMechanism + '\'' +
+                "\n\tsaslCustomConfig='" + saslCustomConfig + '\'' +
```

Comment:
Unless we know that the custom config cannot contain security sensitive information (like a password) then we should mask it:

```suggestion
                "\n\tsaslCustomConfig='" + (saslCustomConfig == null ? null : mask) + '\'' +
```